docs/_examples/integration-tutorial-code/sample-data/test-file.js

```diff
@@ -0,0 +1,32 @@
+/** 
+ * @fileoverview Example data to lint using ESLint. This file contains a variety of errors.
+ * @author Ben Perlmutter
+ */
+v// 'var' should be replaced with 'const' or 'let' (no-var from eslint:recommended)
+var x = 10;
+
+// Unused variable 'y' (no-unused-vars from custom rules)
+const y = 20;
+
+function add(a, b) {
+  // Unexpected console statement (no-console from custom rules)
+  console.log('Adding two numbers');
+  return a + b;
+}
+
+// 'result' is assigned a value but never used (no-unused-vars from custom rules)
+const result = add(x, 5);
+
+// Expected indentation of 2 spaces but found 4 (indent from eslint:recommended)
```

Comment:
Same as above, `indent` is not in `eslint:recommended` and it isn't enabled in the hardcoded configuration inside `example-eslint-integration`.

Reply:
same comment as above. don't remember context. deleting the example